react/src/Toggle/Toggle.tsx

```diff
@@ -19,7 +19,7 @@ export interface ToggleProps extends SwitchProps {
   /**
    * Secondary description text
    */
-  description?: string
+  description?: string | Element
```

Comment:
i think can loosen to `ReactNode`!

Reply:
yea, my bad hahahaha. i didnt even npm ci . will update